src/index/index.js

```diff
@@ -1,16 +1,20 @@
 const express = require("express");
 const morgan = require("morgan");
 const { connectDB } = require("../db");
-const bcrypt = require("bcryptjs");
-const jwt = require("jsonwebtoken");
 
 const app = express();
 
 app.use(morgan("dev"));
-app.use(express.json);
+app.use(express.json());
+
+//base de datos
 connectDB();
 console.log("dataBase connected");
+
+//entrega todos los archivos de este directorio
 app.use(express.static("src"));
-app.listen(6000);
-console.log("server on port, viva la argoyaa", 6000);
+
+//montando servidor
+app.listen(8080);
```

Comment:
app.listen recibe un segundo parametro, que debe ser un callback, el callback sirve para ejecutar algo luego de que el servidor comenzo a andar.

Tambien te conviene tener en una variable el puerto y reutilizarlo en todos los lugares donde necesites usarlo.

```suggestion
const PORT = 8080;
app.listen(PORT, () => {
  console.log("server on port, viva la argoyaa", PORT);
});
```